test/operator.jl

```diff
@@ -529,4 +529,47 @@ using Compat.Test
             @test all(c -> c.set.upper == 3, c7)
         end
     end
+
+    @testset "JuMPArray concatenation" begin
```

Comment:
Are there actually any `JuMPArray`s here? Looks like plain `Array`s to me. The tests name and comments seem out of date. You could then also question why we're testing `vcat` and `hcat` for plain julia `Array`s.

Reply:
> Are there actually any JuMPArrays here?

No, it's two Vector and two Matrix

> You could then also question why we're testing vcat and hcat for plain julia Arrays.

This doubt is supported by the Codecov Report

Reply:
Let's drop these tests then